pandas/tests/extension/test_sparse.py

```diff
@@ -237,6 +237,19 @@ def test_isna(self, data_missing):
     def test_fillna_no_op_returns_copy(self, data, request):
         super().test_fillna_no_op_returns_copy(data)
 
+    def test_fillna_readonly(self, data_missing):
+        # copy=False keyword is not ignored by SparseArray.fillna
```

Comment:
not necessarily for this PR, but should we add an EA attribute specifying whether copy=False is ignored?  that way we could avoid overriding tests?

Reply:
Or could also be an attribute on the Test class

Reply:
yah either way it can be saved for a follow-up